3737/2/commands/start_command.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../helpers/user_helper.rb'
+
+# module with start command
+module StartCommand
+  def start!(*)
+    if UserHelper.registered(from['id'])
+      respond_with :message, text: 'Sorry, but u have already registered!'
+    else
+      respond_with :message, text: 'Enter ur number'
+      save_context :check_user
+    end
+  end
+
+  def check_user(number = nil, *)
+    if UserHelper.check_number(number) && UserHelper.find_user(number).empty?
```

Comment:
check_number - bad naming
what method checks?
user number presence?
such method can be named `exists?` and defined on User model